### DISCUSSION TITLE
Floating Edges and Updatable Edge doesn't work properly

### DISCUSSION THREAD

**Discussion by timburstefan** [xyflow/xyflow#3021] [Q&A]
When I try to connect nodes and edges using easy connect, the updateEdge functionality does not work properly
Does anyone know or if it is possible to give me an example of how to do these two things correctly at the same time?
Thanks.

**Comment by moklick** [comment]
What is not working for you? Could you create a codesandbox for this? https://new.reactflow.dev/react-js

**Reply by timburstefan** [reply]
well, look here
![image](https://user-images.githubusercontent.com/48915514/235652500-aee2569d-bd5d-4368-bdff-f93fa5b12ac5.png)
u can see the floating edges but when i hover the cursor over the edge the 'move' pointer doesnt appear at the end of the edge, instead it appears at the end of the node on the right side like it has default edge connectors.
